src/grpc-server-streaming-method.js

```diff
@@ -27,30 +28,37 @@ class GrpcServerStreamingMethod extends GrpcMethod {
     let request
 
     try {
-      this.logRequestStart()
+      const { method, auth, createLogger, requestOptions } = this
 
-      const { method, auth, logger, requestOptions } = this
+      // generate unique id to be associated with the request
+      const requestId = generateId(6)
```

Comment:
is there a way to reduce the duplication between this and the unary method?

Reply:
By duplication do you mean just the id generation and instantiating the logger?

Reply:
that's the duplication that this PR added, but it's part of some other duplication between the two methods. The answer might be "it's clearer to have duplication" which is what we previously concluded.

Reply:
I do think it's more clear to have this duplication.